aws/platform/variables.tf

```diff
@@ -22,6 +22,12 @@ variable "aws_namespace" {
   description = "Prefix to be applied to created AWS resources"
 }
 
+variable "aws_prometheus_workspace_id" {
```

Comment:
What do you think about accepting an object that matches `prometheus_data_source` above?

`type = object({ host = string, region = string, etc })`

Reply:
I think that works as well. I have now created a `prometheus_data_source` input variable object to pass these values into the module.